### PR TITLE
Cells with Temporary blockers should not exit early in locomotor cache

### LINKED ISSUE #16863: Units can no longer move through gates
Units can no longer path through actors with `ITemporaryBlocker` (mainly TS gates) which causes all units on the TS shellmap to stall.

Temporary blockers need to be taken into account in the locomotor path cache.

### PR DESCRIPTION
**Pull request by teinarss**
Fixes #16863

### CODE REVIEW

**Review by tovl — DISMISSED** [review]
Fix confirmed.

File: OpenRA.Mods.Common/Traits/World/Locomotor.cs [hunk 1]
```diff
@@ -279,7 +279,7 @@ public bool CanMoveFreelyInto(Actor actor, CPos cell, Actor ignoreActor, CellCon
 				return true;
 
 			// Cache doesn't account for ignored actors - these must use the slow path
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Traits/World/Locomotor.cs]: Imho the comment should be updated with a reference to temporary blockers, like `Cache doesn't account for ignored actors or temporary blockers - these must use the slow path`.

**Comment by abcdefg30** [comment]
Fix works. :white_check_mark:

**Comment by teinarss** [comment]
Updated

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/76d05691a29f83e8ee48cf286dc394b55295ff6c...5f17d12aea27083d3f053b96e990c0048bab2aca)

**Pull request merged**